tests/unit/models/transactions/test_did_set.py

```diff
@@ -62,3 +62,46 @@ def test_empty(self):
                 error.exception.args[0],
                 "{'did_set': 'Must have one of `did_document`, `data`, and `uri`.'}",
             )
+
+    def test_all_empty_fields(self):
+        with self.assertRaises(XRPLModelException):
+            DIDSet(account=_ACCOUNT, data="", did_document="", uri="")
+
+    def test_delete_multiple_fields(self):
+        # create a valid DID object
+        tx = DIDSet(
+            account=_ACCOUNT,
+            did_document=_VALID_FIELD,
+            uri=_VALID_FIELD,
+            data=_VALID_FIELD,
+        )
+        self.assertTrue(tx.is_valid())
```

Comment:
This is unnecessary because you're not actually sending the transaction.

Reply:
are you proposing that I remove this entire unit test? Or just the `self.assertTrue(...)` statement?

Reply:
The transaction.